webgpu.h

```diff
@@ -689,6 +689,13 @@ typedef struct WGPUSwapChainDescriptor {
     WGPUPresentMode presentMode;
 } WGPUSwapChainDescriptor;
 
+typedef struct WGPUTextureDataLayout {
```

Comment:
how are we going to ensure these structs are actually nested? i.e. can we just do
```cpp
typedef struct WGPUBufferCopyView {
    WGPUTextureDataLayout layout;
    WGPUBuffer buffer;
} WGPUBufferCopyView;
```

Reply:
That would work. I'm not looking forward to doing another progressive migration :P